app/routes.js

```diff
@@ -5,14 +5,16 @@ const router = express.Router()
 const security = require('./security/index')
 const registration = require('./registration/index')
 const classifyImage = require('./classify_image/index')
+const imageLibraryController = require('./image_library/index')
 
 router.get('/', (request, response) => {
     response.status(200).send({
         'GitHub': 'https://github.com/radiantdinosaurs/slothbucket',
         'Docs': 'https://github.com/radiantdinosaurs/slothbucket#readme'
     })
 })
-router.post('/classify-image', classifyImage.handleClassifyImage)
+router.get('/image-library/:id', security.requiresToken, imageLibraryController.handleImageLibraryRoute)
+router.post('/classify-image', security.requiresToken, classifyImage.handleClassifyImage)
```

Comment:
I know it's the way you've named your modules and feature folders, but personally, I think it's better to follow a simple REST scheme, where each part is a verb or noun depending on the function of the route.

```
GET /images/:id
POST <host>/images/classify
```

Also, it's good practice to version APIs, as in prefix the route with version semantics. This makes it easy to create entirely new APIs and prevent breaking existing consumers.

```
/api/v1/images/classify
```

When you need to add new functionality that would break old functionality, you can just add another route

```
/api/v2/images/classify
```

Older clients can continue consuming v1, but they can move onto v2 at their own pace. This is how a lot of web services are done in order to maintain compatibility